pandas/core/series.py

```diff
@@ -547,6 +547,66 @@ def __len__(self):
         return len(self._data)
 
     def view(self, dtype=None):
+        """
+        Create a new view of the Series.
+
+        This function will return a new Series with a view of the same
+        underlying values in memory, optionally reinterpreted with a new data
+        type. The new data type must preserve the same size in bytes as to not
+        cause index misalignment.
+
+        Series are instantiated with `dtype=float64` by default, so unlike
+        `numpy.array.view()` this function will not try to preserve the current
+        Series data type.
+
+        Parameters
+        ----------
+        dtype : data type
+            Data type object or one of their string representations.
+
+        Returns
+        -------
+        Series
+            A new Series object as a view of the same data in memory.
+
+        See Also
+        --------
+        numpy.ndarray.view : Return a new view of the same data in memory.
```

Comment:
I would say in the explanation here that this is the equivalent numpy method to take a view of an array

Reply:
But they are not *exactly* equivalent because numpy is a little bit more relaxed in this operation since it doesn't need to preserve the amount of data points. But you mean that perhaps I should write:

`numpy.ndarray.view : The equivalent function to create a view of an array.` ?

I think the current way is more formal and clear, *specially* because these functions don't behave exactly alike.

Reply:
> But you mean that perhaps I should write:
>
> numpy.ndarray.view : The equivalent function to create a view of an array. ?

Yes, that is what I meant (I would say "equivalent numpy function" though).  
I think a see also should also give a sense of why we mention the other function. And in this case, because it is the equivalent function of numpy (yes, they don't behave completely the same, but I would keep this for the notes)

Reply:
Makes sense. I'll try to be a bit more explicit on the difference between them in the Notes section.